doc/source/whatsnew/v0.17.1.txt

```diff
@@ -110,6 +110,9 @@ API changes
 
 - ``DataFrame.itertuples()`` now returns ``namedtuple`` objects, when possible. (:issue:`11269`)
 - ``Series.ptp`` will now ignore missing values by default (:issue:`11163`)
+- ``DataFrame.unstack`` and ``Series.unstack`` now take ``fill_value`` keyword to allow direct replacement of
+  missing values when an unstack results in missing values in the resulting ``DataFrame``.  As an added benefit,
+  specifying ``fill_value`` will preserve the data type of the original stacked data.
 
 .. _whatsnew_0171.deprecations:
```

Comment:
add the pr number with an ``:issue:`` tag (IIRC this doesn't have its own dedicated issue)